src/main/webapp/projects/editProject.jsp

```diff
@@ -297,9 +297,9 @@ function addHTMLListeners() {
 
           var res = $.map(data, function(item) {
             console.log("what is in this user el?? "+JSON.stringify(item));
-            if (item.username==myName||typeof item.username == 'undefined'||item.username==undefined||item.username=="") return;
+            if (typeof item.username === 'undefined'||item.username==undefined||item.username==="") return;
```

Comment:
Only admins and users who are already members can see the project edit page. So, removing the self-exclusion here allows admins to add themselves to any project.